jenkins_build

```diff
@@ -4,10 +4,16 @@ set -o errexit -x
 # jenkins job name
 job_name="$1"
 
-export PATH=/usr/local/bin:$PATH
-
-# use local python venv
-. /usr/local/python/bin/activate
+#export PATH=/usr/local/bin:$PATH
+
+## install local python virtual environment
+## based on local requirements.txt
+PYDIR=`pwd`/python
```

Comment:
PYDIR=$(pwd)/python

Reply:
This is a standard recommendation from Codacy's bash script analyses

Reply:
does it show that I learnt my sh programming before that syntax was invented :-D